pysqldb3/pysqldb3.py

```diff
@@ -2103,9 +2108,9 @@ def query_to_gpkg(self, query, path=None, gpkg_name=None, cmd=None, tbl_name = N
         try:
             # Drop the temp table
             if self.type == PG:
-                self.query("drop table {}".format(tmp_table_name), internal=True, strict=False)
+                self.query(f"drop table {tmp_table_name}", internal=True, strict=False)
```

Comment:
i tried to replace the query with `drop_table` but `drop_table` doesn't work since the table in question is a temp table and the function expects a schema